packages/mobile/package.json

```
@@ -32,7 +32,7 @@
     "react-native": "0.69.3",
     "react-native-config": "1.4.6",
     "react-native-mmkv": "2.4.3",
-    "react-native-safe-area-context": "4.3.1",
+    "react-native-safe-area-context": "^4.3.1",
```

We don't use any modifiers.

```suggestion
    "react-native-safe-area-context": "4.3.1",
```